lua/autorun/nl_library.lua

```diff
@@ -0,0 +1,607 @@
+// The tazer is a dependancy at this point //
+AllowedWake = { // these conditions can be removed when a friend presses +use
+	["ZZZ"] = true,
+	["---"] = true,
+	["PLV"] = false
+}
+
+NLL = {}
+
+//NetworkStrings//
+if SERVER then
+	util.AddNetworkString("nllsendhealth")
+	util.AddNetworkString("nllplayercond")
+	util.AddNetworkString("nllupdatebuddha")
+	util.AddNetworkString("nllsetdrowsy")
+
+net.Receive("nllupdatebuddha",function(len,ply)
+	ply.CovertBuddha = net.ReadInt(4)
```

Comment:
if possible move this data attachment stuff to a table in your library instead of attaching onto players directly